docs/sender.md

```diff
@@ -99,8 +99,8 @@ func main() {
         // Set the proxy port to send metrics to. Default: 2878
         MetricsPort : 2878, 
         
-        // Set a proxy port to send histograms to.  Recommended: 40000
-        DistributionPort: 40000,
+        // Set a proxy port to send histograms to.  Recommended: 2878
+        DistributionPort: 2878,
         
         // Set a proxy port to send trace data to. Recommended: 30000
         TracingPort : 50000,
```

Comment:
@susanjlindner this should be 30,000
TracingPort : 30000,

Reply:
Happy to change it to 30000. I wasn't sure whether Vikram intended to show a different number being used in the example.